HLTrigger/Timer/plugins/FastTimerServiceClient.cc

```diff
@@ -353,12 +352,12 @@ void FastTimerServiceClient::fillPathSummaryPlots(DQMStore::IBooker& booker,
     }
 
     // vs lumi
-    if (doPlotsVsScalLumi_)
-      fillPlotsVsLumi(booker, getter, subsubdir, "VsScalLumi", scalLumiMEPSet_);
+    if (doPlotsVsOnlineLumi_)
+      fillPlotsVsLumi(booker, getter, subsubdir, "vs_lumi", onlineLumiMEPSet_);
```

Comment:
```suggestion
      fillPlotsVsLumi(booker, getter, subsubdir, "_vs_lumi", onlineLumiMEPSet_);
```

and so on for the others

Reply:
Mh.. but one underscore is added [here](https://github.com/cms-sw/cmssw/blob/e35468fc103bb4765cf68f830e48f1427c5d6330/HLTrigger/Timer/plugins/FastTimerServiceClient.cc#L435) (I also added a screenshot to the description). Or do we really want the double underscore?